site/assets/scss/_sidebar.scss

```
@@ -41,28 +42,41 @@
       text-decoration: if($link-hover-decoration == underline, none, null);
       background-color: rgba($bd-purple-bright, .1);
     }
+  }
+
+  .btn {
+    // Custom styles (as we don't have a completely neutral button style)
+    padding: .25rem .5rem;
+    font-weight: 600;
+    color: rgba($black, .65);
+    background-color: none;
+    border: none;
+
+    &:hover,
+    &:focus {
+      color: rgba($black, .85);
+      background-color: rgba($bd-purple-bright, .1);
+    }
 
-    // Indent if there's no submenu
-    &:only-child {
-      margin-left: 1.25rem;
+    &:focus {
+      box-shadow: 0 0 0 1px rgba($black, .45);
```

Can this be themed like the rest? The black looks off to me.

```suggestion
      box-shadow: 0 0 0 1px rgba($bd-purple-bright, .25);
```

---

if this leads to a sufficient contrast of at least 3:1 for the outline, sure. let me check (as it's all variables within variables, it's difficult to ascertain without actually running it)

---

ok, `.25` is insufficient to reach 3:1 ratio. `.7` (technically, `.69`) was the cleanest one that got us just over the 3:1 mark.

---

(and yes, as mentioned previously elsewhere, our button focus styles for `.btn-primary` etc are well below the 3:1 ratio as well, something that we'll need to look into. and again, even more reason for something related to `:focus-visible` to allow us strong styles for keyboard users and light-touch ones for mouse/touch users)